Client/Frontend/Autofill/CreditCard/CreditCardSettingsViews+Model/CreditCardTableViewModel.swift

```diff
@@ -0,0 +1,39 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import SwiftUI
+import Storage
+import Shared
+
+class CreditCardTableViewModel {
+    var creditCards: [CreditCard] = [CreditCard]() {
+        didSet {
+            didUpdadteCreditCards?()
+        }
+    }
+
+    var didUpdadteCreditCards: (() -> Void)?
+
+    var autofillCreditCardStatus: Bool {
```

Comment:
I find this variable name confusing: what does status stand for? Does that represent if autofill is enabled? In that case I'd suggest to rename it to `isAutofillEnabled`.